common/journal_entries/imperia_carlist_war_je.txt

```diff
@@ -0,0 +1,69 @@
+﻿je_carlist_war = {
+	icon = "gfx/interface/icons/event_icons/waving_flag.dds"
+
+	group = je_group_foreign_affairs
+
+	should_be_pinned_by_default = yes
+
+	modifiers_while_active = {
+		modifier_carlist_war_funding
+	}
+
+	complete = {
+		is_at_war = no
+		owns_entire_state_region = STATE_TOLEDO
+	}
```

Comment:
I would rather that you check for the existence of Spain. Especially since you implement the actual consequence by changing the tag to Spain.